source/misc/customcitizennames.md

```diff
@@ -32,7 +32,7 @@ Everything you have to change is just one file.
 <br><br>
 
 ## Official language templates
-For download the file click the _raw_ button on top of the page and then right-click -> save as...<br>
+
```

Comment:
```suggestion
If the file doesn't automatically download, try clicking the *raw* button on the top of the page and then right-clicking 'save as'.
```

Reply:
just in case

Reply:
I think it downloads directly in every browser, but we can write it like this if you want.

Reply:
yeah, just in case. probably redundant though